api/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
 
 # Create your models here.
```

Comment:
Can you remove this comment as it is not needed. 

We want to come out like a real pros :)